def additoin(x, y):
    try:
        x = 10
        y = 20
        print("Addition:", x + b)
    except Exception as e:
        print(e.__class__)
        print("Something went wrong")
    else:
        print("the operation is successful")
additoin(10, 20)
